src/main/webapp/app/exam/participate/summary/points-summary/exam-points-summary.component.html

```diff
@@ -20,10 +20,10 @@ <h4>{{ 'artemisApp.exam.examSummary.points.overview' | translate }}</h4>
                     {{ calculateAchievedPoints(exercise) }}
                 </td>
                 <td>
-                    <span *ngIf="exercise.maxScore">{{ exercise.maxScore }}</span>
+                    <span *ngIf="exercise.maxScore != undefined">{{ exercise.maxScore }}</span>
```

Comment:
I might be wrong but how about we change that: `*ngIf="exercise.maxScore"`? The check would cover not only null/undefined but also wouldn't display if max score is 0